### DISCUSSION TITLE
How to remove Announced LabelSets when qurey from Thanos-query UI

### DISCUSSION THREAD

**Discussion by yccharles** [thanos-io/thanos#5304] [Questions & Answers]
hi, guys . When I qurey from thanons-query UI , all response data would be adding label "prometheus="monitoring/k8s"" automatically。 

How can I remove  the Announced LabelSets  just like using prometheus ui .

Beacuse I meet a case:  my code would be querying metric from prometheus. but now,  we use thanos-query instead of prometheus.  The metric below is not working anymore. 
```
 :node_cpu_utilisation:avg1m * sum(node:node_num_cpu:sum) 
```

I know It would be working  if I change my metric like this.  But Is there any way to remove the label "prometheus="monitoring/k8s" from thanos-query response data?  This way I can using Thanos-query platform without change my code anymore.
```
 :node_cpu_utilisation:avg1m * on () sum(node:node_num_cpu:sum) 
```

![image](https://user-images.githubusercontent.com/50130457/165464870-4b00de29-622b-4f49-83c0-cb748b5415ba.png)

**Comment by yccharles** [comment]
@GiedriusS look forward for your help.

**Reply by artemkrim** [reply]
solution?
